airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/GlueOperations.java

```diff
@@ -0,0 +1,156 @@
+            awsGlueClient.getTable(getTableRequest);
+
+            UpdateTableRequest updateTableRequest = new UpdateTableRequest()
+                .withDatabaseName(databaseName)
+                //TODO (itaseski) do we need to send all table inputs or the ones that are going to be changed?
+                .withTableInput(
+                    new TableInput()
+                        .withName(tableName)
+                        .withStorageDescriptor(
+                            new StorageDescriptor()
+                                .withLocation(location)
+                                .withColumns(transformSchema(jsonSchema))
+                        )
+                );
+
+            awsGlueClient.updateTable(updateTableRequest);
+        } catch (EntityNotFoundException enfe) {
+            CreateTableRequest createTableRequest = new CreateTableRequest()
+                .withDatabaseName(databaseName)
+                .withTableInput(
+                    new TableInput()
+                        .withName(tableName)
+                        .withTableType("GOVERNED")
+                        .withStorageDescriptor(
+                            new StorageDescriptor()
+                                .withLocation(location)
+                                .withColumns(transformSchema(jsonSchema))
+                                .withInputFormat("org.apache.hadoop.mapred.TextInputFormat")
+                                .withOutputFormat("org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat")
+                                .withSerdeInfo(
+                                    new SerDeInfo()
+                                        .withSerializationLibrary("org.openx.data.jsonserde.JsonSerDe")
+                                        .withParameters(Map.of("paths", ","))
+                                )
+                        )
+                        .withPartitionKeys(List.of())
+                        .withParameters(Map.of("classification", "json"))
+                );
+
+            awsGlueClient.createTable(createTableRequest);
+        }
+    }
+
+    @Override
+    public void deleteTable(String databaseName, String tableName) {
+
+        DeleteTableRequest deleteTableRequest = new DeleteTableRequest()
+            .withDatabaseName(databaseName)
+            .withName(tableName);
+
+        awsGlueClient.deleteTable(deleteTableRequest);
+
+    }
+
+    public Collection<Column> transformSchema(JsonNode jsonSchema) {
+
+        UnaryOperator<String> typeMapper = type -> switch (type) {
+            case "string" -> "string";
+            case "number" -> "float";
+            // TODO (itaseski) does the airbyte schema support integers as first class types or as airbyte_types?
+            case "integer" -> "int";
+            case "boolean" -> "boolean";
+            // TODO (itaseski) throw exception on unknown types or set as received?
```

Comment:
I think it's fine to keep them as-is (at least as a first pass)

Reply:
Kept as a default case which will send the type as received without performing any mapping on it.